src/main/java/io/codelex/flightplanner/service/FlightService.java

```diff
@@ -40,41 +51,33 @@ public Flight addFlight(AddFlightRequest flightRequest) {
         if (!isArrivalAndDepartureTimeValid(flightRequest)) {
             throw new ResponseStatusException(HttpStatus.BAD_REQUEST);
         }
-        if (flightExists(flightRequest)) {
-            throw new ResponseStatusException(HttpStatus.CONFLICT);
-        }
-        airportService.add(airportFrom);
-        airportService.add(airportTo);
         Flight flight = Flight.builder()
```

Comment:
es builderi šeit gan neņemtu, vienmēr būs visi lauki izmantoti, builderi drīzak izmantot, kad ir daudz optional lauki

Reply:
Manā skatījumā builder smukāk uzrakstās nekā garš konstruktors ar miljons parametriem.

Reply:
:smile: 
atleast restrict the access thne, otherwise feels quite fuck up prone

Reply:
Es visdrīzāk tuvākajās dienās no Builder atteikšos.

Reply:
as long as you know the difference more than enough by now :+1: